MobileSyncExplorerSwift/RecentContacts/RecentContactsWidget.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  RecentContactsWidget.swift
+//  RecentContacts
+//
+//  Created by Brianna Birman on 1/21/22.
+//  Copyright © 2022 MobileSyncExplorerSwiftOrganizationName. All rights reserved.
+//
+
+import WidgetKit
+import SwiftUI
+
+struct Provider: TimelineProvider {
+    func placeholder(in context: Context) -> ContactsEntry {
+        let contacts = [ContactSummary(id: "1", firstName: "Michelle", lastName: "Kim"),
```

Comment:
What are the hardcoded values for?

Reply:
The placeholder part will show a redacted version of this view as a loading stencil so it won't actually show the data, and then snapshot is also using it. Snapshot is used for the discovery view (screenshot below), so this will show even if the user isn't logged in, but it'd also be nice to show real data when they are logged in, could add a check for that
<img width="564" alt="Screen Shot 2022-02-07 at 10 54 25 AM" src="https://user-images.githubusercontent.com/840431/152855389-6a76d90e-b9b0-43f7-88ad-97ee5ebae969.png">